### ISSUE TITLE
Bug: UniFi Pro Max 'Etherlighting' LEDs not detected in Home Assistant

### ISSUE THREAD

**Issue by sirkirby** [sirkirby/unifi-network-rules#96]
### Describe the bug
LED switch feature does not work with the new **UniFi Pro Max** line that includes *Etherlighting* ports. Devices do **not** appear under Devices & Services in Home Assistant and no LED entities are created.

### To Reproduce
1. Install UniFi Network Rules integration (v3.2.0) on Home Assistant 2025.7.1
2. Add a UniFi Pro Max switch (firmware 9.1.0 or later) to the controller.
3. Reload the integration or call `unifi_network_rules.refresh_rules`.

### Expected behavior
The Etherlighting LED ports should surface as switch entities, similar to standard UniFi switch LEDs.

### Actual behavior
No devices/entities for the Pro Max switch appear; LED control is impossible.

### Logs
```text
2025-08-07 10:12:05 DEBUG custom_components.unifi_network_rules: Device LED discovery found 0 LED-capable ports
2025-08-07 10:12:05 WARNING custom_components.unifi_network_rules: No LED-capable devices discovered; skipping LED entity setup
```

### Environment
- Integration version: 3.2.0
- Home Assistant: 2025.7.1
- UniFi Network Application: 9.1.0
- UniFi device: Pro Max 48-PoE (firmware 9.1.0)

### Additional context
Early reports suggest the Pro Max uses new LED API endpoints. Investigation needed to add support and surface the entities.
